### PR TITLE
ACMS-644: Enable Drupal 9 & 10 compatible release of GDPR with securi…

### PR DESCRIPTION
**Pull request by chandan-singh7929**
…ty coverage.

### CODE REVIEW

**Review by rajeshreeputra — APPROVED** [review]
Changes Looks Good.

File: acms/acms.yml [hunk 1]
```diff
@@ -101,6 +101,16 @@ questions:
           - no
       skip_on_value: false
       default_value: "no"
+    gdpr_integration:
+      dependencies:
+        starter_kits: acquia_cms_enterprise_low_code || acquia_cms_community
+      question: "Would you like to add GDPR functionality to the site ?"
```
Comment (apathak18) [inline acms/acms.yml:107]: Replace with `Would you like to add GDPR functionality to the site (Yes/No) ?`

Comment (chandan-singh7929) [reply acms/acms.yml:107]: @vishalkhode1 - what do you suggest?

Comment (chandan-singh7929) [reply acms/acms.yml]: Agreed, update the question text as per suggestion.

**Review by deepakmishra2 — APPROVED** [review]

**Pull request merged**